test/integration/join-test.js

```diff
@@ -108,6 +108,22 @@ testRingpopCluster({
 });
 
 testRingpopCluster({
+    tap: function tap(cluster) {
+        cluster.forEach(function eachNode(node) {
+            node.gossip.start = function noop() {};
+        });
+    },
+    waitForConvergence: false
+}, 'do not disseminate join list', function t(bootRes, cluster, assert) {
+    cluster.forEach(function eachNode(node) {
+        var changesKeys = Object.keys(node.dissemination.changes);
```

Comment:
Instead of reaching into the internal state of the dissemination object and pulling out the changes, expose an API so you can call:

```
dissemination.getChangesCount();
dissemination.getChangeAt(0);
```

This helps protect having to modify tests if we were rename the member variable of the dissemination object or refactor in some other way.